core/services/relay/evm/functions/logpoller_wrapper.go

```diff
@@ -47,18 +61,44 @@ func NewLogPollerWrapper(routerContractAddress common.Address, pluginConfig conf
 	}
 	blockOffset := int64(pluginConfig.MinIncomingConfirmations) - 1
 	if blockOffset < 0 {
+		lggr.Warnw("invalid minIncomingConfirmations, using 1 instead", "minIncomingConfirmations", pluginConfig.MinIncomingConfirmations)
 		blockOffset = 0
 	}
+	requestBlockOffset := int64(pluginConfig.MinRequestConfirmations) - 1
+	if requestBlockOffset < 0 {
+		lggr.Warnw("invalid minRequestConfirmations, using minIncomingConfirmations instead", "minRequestConfirmations", pluginConfig.MinRequestConfirmations)
+		requestBlockOffset = blockOffset
+	}
+	responseBlockOffset := int64(pluginConfig.MinResponseConfirmations) - 1
+	if responseBlockOffset < 0 {
+		lggr.Warnw("invalid minResponseConfirmations, using minIncomingConfirmations instead", "minResponseConfirmations", pluginConfig.MinResponseConfirmations)
+		responseBlockOffset = blockOffset
+	}
+	logPollerCacheDurationSec := int64(pluginConfig.LogPollerCacheDurationSec)
+	if logPollerCacheDurationSec <= 0 {
+		lggr.Warnw("invalid logPollerCacheDuration, using 300 instead", "logPollerCacheDurationSec", logPollerCacheDurationSec)
+		logPollerCacheDurationSec = 300
```

Comment:
const

Reply:
Can you explain what you mean by `const` here? I'm not understanding why this should be a const variable.

Reply:
Bolek probably means because this is a magic number we should instead set it to a more descriptive named const such as `const PastBlocksToPoll = 50`. My personal preferenece would also be to pull this either the top of the file or to a `constants` area for easy reference and changing.